repositories/user.repository.js

```diff
@@ -93,6 +93,52 @@ const deleteUserByUsername = async (username) => {
   }
 };
 
+const userSignUp = async (
+  id,
+  username,
+  email,
+  password,
+  createdAt,
+  updatedAt
+) => {
+  try {
+    const user = await User.create({
+      id: id,
+      username: username,
+      email: email,
+      password: password,
+      createdAt: createdAt,
+      updatedAt: updatedAt,
+    });
+    return user;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const userLogIn = async (
+  username
+  ) => {
+  try {
+    const user = await User.findOne({
+      where: {
+        username: username,
+      },
+    });
+
+    if (user) {
+      const dtoUser = new UserDTO(user);
+      return dtoUser;
+    } else {
+      return null;
+    }
+  } catch (error) {
+    throw error;
+  }
+};
```

Comment:
this should be userFindByUsername, you are not creating any entry for login, you are just checking if the user exists.

Reply:
resolved